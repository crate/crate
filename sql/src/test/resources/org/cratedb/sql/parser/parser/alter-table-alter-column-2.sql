ALTER TABLE t1 ALTER COLUMN c NOT NULL