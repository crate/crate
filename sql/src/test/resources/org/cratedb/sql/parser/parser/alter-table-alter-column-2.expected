org.cratedb.sql.parser.parser.AlterTableNode@7d53ccbe
name: t1
statementType: ALTER TABLE
objectName: t1
compressTable: false
sequential: false
truncateTable: false
purge: false
defragment: false
truncateEndOfTable: false
updateStatistics: false
updateStatisticsAll: false
indexNameForUpdateStatistics: null
tableElementList: 	
	org.cratedb.sql.parser.parser.TableElementList@8a70e3a

	[0]:	
	org.cratedb.sql.parser.parser.ModifyColumnNode@f970646
	type: null
	name: c
	elementType: AT_MODIFY_COLUMN